common.bazelrc

```diff
@@ -76,6 +76,9 @@ common --experimental_allow_incremental_repository_updates
 build --incompatible_strict_action_env
 run --incompatible_strict_action_env
 
+# when running `bazel coverage` ensure that the test targets are instrumented 
```

Comment:
can you expand on this? what changes/breaks if it is missing? is this something users will need to add downstream?

Reply:
added a better description, what do ya think?
```
# When running `bazel coverage` --instrument_test_targets needs to be set in order to 
# collect coverage information from test targets
coverage --instrument_test_targets
` ``